### ISSUE TITLE
Tinker's Fortification upgrades are buggy.

### ISSUE THREAD

**Issue by Roflpwnage** [NillerMedDild/Enigmatica2Expert#1421]
Enigmatica2Expert
Version 1.71

**Issue**
TL;DR: Applying a diamond modifier to a tool, before fortification, causes all added fortifications to be 1 mining level lower than it should be.

Additional Testing(I only tested pickaxes):: This seems to only apply to pickaxe heads with mining levels of certus or lower (certus, copper, stone/flint) [An iron pickaxe head or greater is unaffected]. If you level a tool with a bugged fortification, it seems to update correctly, but that can get expensive as it doesn't completely fix the problem, as you would need to level the tool everytime you apply a new fortification.

In other words; if you fortify a, lets say, a stone head pickaxe that already has a diamond on it, with obsidian, it will only be able to mine osmium level materials. cobalt would be obsidian.

**Comment by NillerMedDild** [comment]
Thank you for the report, I will investigate the issue.

**Comment by NillerMedDild** [comment]
I'm going to write this off as an added cost of using cheaper tool heads :)
